apps/test/unit/templates/lessonOverview/LessonNavigationDropdownTest.jsx

```diff
@@ -0,0 +1,106 @@
+};
+
+describe('LessonNavigationDropdown', () => {
+  it('renders dropdown for short lesson list', () => {
+    lesson.unit.lessons = shortLessonList;
+    const wrapper = shallow(<LessonNavigationDropdown lesson={lesson} />);
+    expect(wrapper.find('DropdownButton').length).to.equal(1);
+    expect(wrapper.find('a').length).to.equal(2);
+
+    expect(wrapper.contains('1 - Lesson 1')).to.be.true;
+    expect(wrapper.contains('2 - Lesson 2')).to.be.true;
+  });
+
+  it('renders dropdown for long lesson list', () => {
+    lesson.unit.lessons = longLessonList;
+    const wrapper = shallow(<LessonNavigationDropdown lesson={lesson} />);
+    expect(wrapper.find('DropdownButton').length).to.equal(1);
+    expect(wrapper.find('a').length).to.equal(12);
+
+    expect(wrapper.contains('Lessons 1 to 10')).to.be.true;
+    expect(wrapper.contains('1 - Lesson 1')).to.be.true;
+    expect(wrapper.contains('2 - Lesson 2')).to.be.true;
+    expect(wrapper.contains('3 - Lesson 3')).to.be.true;
+    expect(wrapper.contains('4 - Lesson 4')).to.be.true;
+    expect(wrapper.contains('5 - Lesson 5')).to.be.true;
+    expect(wrapper.contains('6 - Lesson 6')).to.be.true;
+    expect(wrapper.contains('7 - Lesson 7')).to.be.true;
+    expect(wrapper.contains('8 - Lesson 8')).to.be.true;
+    expect(wrapper.contains('9 - Lesson 9')).to.be.true;
+    expect(wrapper.contains('10 - Lesson 10')).to.be.true;
+    expect(wrapper.contains('Lessons 11 to 15')).to.be.true;
+  });
+
+  it('switches open section when click new section', () => {
+    lesson.unit.lessons = longLessonList;
+    const wrapper = shallow(<LessonNavigationDropdown lesson={lesson} />);
+    expect(wrapper.find('a').length).to.equal(12);
+
+    expect(wrapper.contains('Lessons 1 to 10')).to.be.true;
+    expect(wrapper.contains('1 - Lesson 1')).to.be.true;
+    expect(wrapper.contains('2 - Lesson 2')).to.be.true;
+    expect(wrapper.contains('3 - Lesson 3')).to.be.true;
+    expect(wrapper.contains('4 - Lesson 4')).to.be.true;
+    expect(wrapper.contains('5 - Lesson 5')).to.be.true;
+    expect(wrapper.contains('6 - Lesson 6')).to.be.true;
+    expect(wrapper.contains('7 - Lesson 7')).to.be.true;
+    expect(wrapper.contains('8 - Lesson 8')).to.be.true;
+    expect(wrapper.contains('9 - Lesson 9')).to.be.true;
+    expect(wrapper.contains('10 - Lesson 10')).to.be.true;
+    expect(wrapper.contains('Lessons 11 to 15')).to.be.true;
+
+    let section2 = wrapper.find('a').at(11);
+    section2.simulate('click');
+
+    expect(wrapper.find('a').length).to.equal(7);
+
+    expect(wrapper.contains('Lessons 1 to 10')).to.be.true;
+    expect(wrapper.contains('Lessons 11 to 15')).to.be.true;
+    expect(wrapper.contains('11 - Lesson 11')).to.be.true;
+    expect(wrapper.contains('12 - Lesson 12')).to.be.true;
+    expect(wrapper.contains('13 - Lesson 13')).to.be.true;
+    expect(wrapper.contains('14 - Lesson 14')).to.be.true;
+    expect(wrapper.contains('15 - Lesson 15')).to.be.true;
+  });
```

Comment:
I would recommend adding a unit test showing that navigation occurs when you click a 'navigate' entry, using the wrapper method mentioned earlier. the reason I'm suggesting this is because you have some logic in this component around when clicking should navigate or not.